src/server/v2.0/handler/project.go

```diff
@@ -147,6 +147,13 @@ func (a *projectAPI) CreateProject(ctx context.Context, params operation.CreateP
 		req.Metadata.Public = strconv.FormatBool(false)
 	}
 
+	// validate metadata.public value, should only be "true" or "false"
```

Comment:
use type assertion?

Reply:
we only support two value, true or false.